src/components/Dropdown.js

```diff
@@ -23,18 +23,8 @@ const StyledCard = styled(motion.custom(Card))`
   z-index: ${props => props.zIndex};
   overflow-y: auto;
   max-height: 124px;
-  left: ${props =>
-    props.stretched || props.align === 'left'
-      ? `${0 + props.offsetSide}px`
-      : 'auto'};
-  right: ${props =>
-    props.stretched || props.align === 'right'
-      ? `${0 + props.offsetSide}px`
-      : 'auto'};
-  top: ${props =>
-    `${spacing.padding.big + spacing.padding.tiny + props.offsetTop}px`};
   padding: 12px ${spacing.padding.small}px;
-  max-width: ${props => (props.stretched ? 'auto' : '172px')};
+  width: 100%;
```

Comment:
Shouldn't the `172px` max still be passed to the portalConfig if stretched is false?

Reply:
The react relative dropdown component calculates the max width based on the component that it wraps. This basically negates the need for us to have a hard coded max-width in the component.